.github/workflows/ci.yaml

```diff
@@ -134,3 +134,22 @@ jobs:
         push: false
         cache-from: type=gha
         cache-to: type=gha,mode=max
+
+  build-cli:
+    needs: [test-unit, lint-go, lint-charts, lint-proto, check-codegen]
+    runs-on: ubuntu-latest
+    container:
+      image: golang:1.20.4-bullseye    
```

Comment:
golang:1.20.7-bookworm

Reply:
Oh. Or actually not, since this is a new job. Good catch.

Reply:
Fixed. Thanks again.